api/server.go

```diff
@@ -305,6 +305,8 @@ func RunServer(dry bool) http.Handler {
 	m.Add("1.2", http.MethodGet, "/apps/{app}/certificate", AuthorizationRequiredHandler(listCertificates))
 	m.Add("1.2", http.MethodPut, "/apps/{app}/certificate", AuthorizationRequiredHandler(setCertificate))
 	m.Add("1.2", http.MethodDelete, "/apps/{app}/certificate", AuthorizationRequiredHandler(unsetCertificate))
+	m.Add("1.0", http.MethodPut, "/apps/{app}/certissuer", AuthorizationRequiredHandler(setCertIssuer))
```

Comment:
@gvicentin we may change "1.0" to future version of tsuru api.

Reply:
Thanks! Fixed ✅